### PR TITLE
Reduce ipa dependencies

### PR DESCRIPTION
**Pull request by mrinnetmaki**
Add *generic* profiles for `Medication` and `MedicationRequest` for Kela's use case where they cannot use the IPA dependent profiles (because `Medication.code` is mandatory there, and in Kela's use case they may not have it available).

### CODE REVIEW

File: input/fsh/practitioner.fsh [hunk 1]
```diff
@@ -4,7 +4,6 @@ Id: fi-base-practitioner
 Title: "FI Base Practitioner"
 Description: "This is the Finnish base profile for the Practitioner resource."
 * ^status = #draft
-* id 1..1
```
Comment (mikajylha) [inline input/fsh/practitioner.fsh:7]: 👍

File: input/pagecontent/StructureDefinition-fi-base-patient-medication-request-intro.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### Scope and Usage
+
+The FI Base MedicationRequest derives from the MedicationRequest profile of the International
```
Comment (mikajylha) [inline input/pagecontent/StructureDefinition-fi-base-patient-medication-request-intro.md:3]: FI Base Patient MedicationRequest

**Review by mikajylha — CHANGES_REQUESTED** [review]
Patient medication request refers wrong resource in Scope and usage.

File: input/pagecontent/StructureDefinition-fi-base-patient-medication-request-intro.md (hunk 2 above)
Comment (teropekkola) [reply input/pagecontent/StructureDefinition-fi-base-patient-medication-request-intro.md:3]: Fixed in branch "descriptions" (QA fine tunings).

**Pull request merged**